plugin/output/splunk/splunk.go

```diff
@@ -129,46 +143,45 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 	root := insaneJSON.Spawn()
 	outBuf := data.outBuf[:0]
 	for _, event := range batch.Events {
-		root.AddField("event").MutateToNode(event.Root.Node)
+		for _, field := range event.Root.AsFields() {
+			key := field.AsString()
+			val := field.AsFieldValue()
+			if key == p.config.EventKey {
+				node := root.AddField("event").MutateToObject()
+				node.AddField(key).MutateToNode(val)
+				continue
+			}
+			root.AddField(key).MutateToNode(val)
```

Comment:
it's already in the root. why does it need to add the same field one more time?

Reply:
root is empty as per https://github.com/ozonru/file.d/pull/41/files#diff-b7d3945688c3a827787b7eeef8303aa7e0f7e5fe26fed4108068ad34f77e3301R143

it has been extracted before, too, but added to "event" key, now it is added as separate key